CRM/Case/XMLProcessor/Process.php

```diff
@@ -88,7 +88,7 @@ public function process($xml, &$params) {
       // create relationships for the ones that are required
       foreach ($xml->CaseRoles as $caseRoleXML) {
         foreach ($caseRoleXML->RelationshipType as $relationshipTypeXML) {
-          if ((int ) $relationshipTypeXML->creator == 1) {
+          if ((int) $relationshipTypeXML->creator == 1) {
```

Comment:
This cast is also of dubious utility. Hard to tell if it gets acted upon before or after the `==` comparison, but:
- If before, then the `==` operator is not strict about type, so the cast is unnecessary.
- If after, the `if()` statement will cast to boolean so the cast to int doesn't matter.

Reply:
For that matter, `creator` looks like a boolean field anyway, so I think both the `(int)` cast **and** the `== 1` are unnecessary.

Reply:
I was curious about why it didn't give undefined property notices if there was no creator member. Seems these SimpleXML objects function slightly differently, so it's not needed to check if it exists. So, good.